### ISSUE TITLE
wpg-install broke my i3 config

### ISSUE THREAD

**Issue by alizeegagnon** [deviantfero/wpgtk-templates#21]
I know it was clear that this would replace my current config, but I don't understand why it has to change so many keyboard shortcuts, like closing a window, opening rofi... Also, it added some commands that are specific to i3-gaps, which I am not using. I was expecting to receive the basic i3 config and not something that feels like someone else's riced config.

Is this expected behavior ?

**Comment by deviantfero** [comment]
yeah, it is, sorry about this, you should have a backup though, the script creates a backup of your original file before replacing it, I'll leave this issue open because I should really just provide something more basic like what you're saying.

the backup should be located in `~/.config/i3/config.bak`
